prow/plugins/lifecycle/close_test.go

```diff
@@ -157,9 +170,31 @@ func TestCloseComment(t *testing.T) {
 			shouldComment: false,
 			shouldAssign:  true,
 		},
+		{
+			name:          "close by other person, stale issue",
+			action:        github.GenericCommentActionCreated,
+			state:         "open",
+			body:          "/close",
+			commenter:     "non-member",
+			labels:        []string{"lifecycle/stale"},
+			shouldClose:   true,
+			shouldComment: false,
+			shouldAssign:  false,
```

Comment:
`shouldAssign` is not used.

Reply:
Not sure what you mean here... It is definitely used later in the function.

Reply:
You're right, I misread the diff.